.github/workflows/nbval.yml

```diff
@@ -188,7 +188,7 @@ jobs:
       if: runner.os == 'MacOS'
       run: |
         chmod +x .ci/validate_notebooks.sh
-        .ci/validate_notebooks.sh 107-speech-recognition-quantization 114-quantization-simplified-mode 115-async-api
+        .ci/validate_notebooks.sh 107-speech-recognition-quantization 114-quantization-simplified-mode 115-async-api 203-meter-reader 219-knowledge-graphs-conve
```

Comment:
You're ignoring these notebooks only for mac, not for all oses. Look at line 188

Reply:
Thanks for spotting that. I will include it for OS: Windows and Linux as well.